roles/ovirt-cluster-upgrade/README.md

```diff
@@ -20,6 +20,7 @@ Role Variables
 | upgrade_timeout         | 1200                  | Timeout in seconds to wait for host to be upgraded. |
 | host_statuses           | [UP]                  | List of host statuses. If a host is in any of the specified statuses then it will be upgraded. |
 | host_names              | [\*]                  | List of host names to be upgraded.        |
+| check_upgrade           | false                 | If true, run check_for_upgrade action on all hosts before trying to upgrade them. If false, we only upgrade hosts which reporsts available updates. |
```

Comment:
If `true`, run check_for_upgrade action on all hosts before executing upgrade on them. If `false`, run upgrade only for hosts with available upgrades and ignore all other hosts.

Reply:
done